java/src/com/ibm/streamsx/topology/internal/context/ToolkitStreamsContext.java

```diff
@@ -232,7 +232,8 @@ public FileVisitResult preVisitDirectory(final Path dir,
             public FileVisitResult visitFile(final Path file,
                     final BasicFileAttributes attrs) throws IOException {
                 Files.copy(file,
-                        targetPath.resolve(sourcePath.relativize(file)));
+                        targetPath.resolve(sourcePath.relativize(file)),
+                        StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Due to namespace packages, two directories can be merged (both copied into `opt/python/packages).  This change is needed if two directories contain a conflicting file name.  Obviously only one file can exist in the target destination; the other is overwritten.  The file that wins should be the module that is actually loaded (comes first in the Python search path).